packages/lesswrong/components/posts/TableOfContents/TableOfContentsList.tsx

```diff
@@ -12,15 +12,37 @@ const isRegularClick = (ev: React.MouseEvent) => {
   return ev.button===0 && !ev.ctrlKey && !ev.shiftKey && !ev.altKey && !ev.metaKey;
 }
 
+const sectionsWithAnswersSorted = (sections: ToCSection[], sorting: "newest" | "oldest") => {
```

Comment:
This way of sorting the answers in the ToC is a fragile hack, it might be better to just not do it and keep the default sorting, but I'll leave it to you.

Reply:
> Maybe you can make anchor a union type?

The anchor can be the answer id or some text generated from a heading, e.g. if you have a heading with the text "answers" it will have the same anchor as the "Answers" text in the ToC.

In any case, thinking again about it, we don't even need to find those two "separators", I changed the function to just sort the answers and keep the other sections in their original locations, curious what you think!

> You seem good with TS

I wish... https://github.com/sno2/bf#example https://github.com/fc01/TypeGame